parachain/runtime/paseo/src/governance_v2/tracks.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020-2024 Trust Computing GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+//! Track configurations for governance.
+
+use super::*;
+use runtime_common::currency::DOLLARS;
+
+const fn percent(x: i32) -> sp_runtime::FixedI64 {
+	sp_runtime::FixedI64::from_rational(x as u128, 100)
+}
+const fn permill(x: i32) -> sp_runtime::FixedI64 {
+	sp_runtime::FixedI64::from_rational(x as u128, 1000)
+}
+
+use pallet_referenda::Curve;
+const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 4] = [
+	(
+		0,
+		pallet_referenda::TrackInfo {
+			// Name of this track.
+			name: "root",
+			// A limit for the number of referenda on this track that can be being decided at once.
+			// For Root origin this should generally be just one.
+			max_deciding: 5,
```

Comment:
Why 5 on the root track? Shouldn’t this be 1?

Reply:
I just think in the past time we had multiple root operations simultaneously.
But you are correct, 1 will more fit DAO concept. Maybe we can rediscuss the parameter when impl it on mainnet.